server.go

```diff
@@ -488,3 +499,59 @@ func getClaText(claTextUrl string) (claText string, err error) {
 
 	return claCache[claTextUrl], nil
 }
+
+const envSmtpHost = "SMTP_HOST"
+const envSmtpPort = "SMTP_PORT"
+const envSmtpUsername = "SMTP_USERNAME"
+const envSmtpPassword = "SMTP_PASSWORD"
+const envNotificationAddress = "NOTIFY_EMAIL"
+
+func handleTestEmail(c echo.Context) (err error) {
+	testSignature := new(types.UserSignature)
+	testSignature.User.Login = "LOGIN-ID"
+	testSignature.User.Email = "someone@somewhere.tld"
+	testSignature.User.GivenName = "A Person"
+	testSignature.CLAVersion = getCurrentCLAVersion()
+	testSignature.TimeSigned = time.Now()
+	testSignature.CLATextUrl = os.Getenv(envClsUrl)
+	testSignature.CLAText, _ = getClaText(testSignature.CLATextUrl)
+
+	return notifySignatureComplete(testSignature)
+}
+
+func notifySignatureComplete(signature *types.UserSignature) (err error) {
```

Comment:
My +1 still holds.

I don't consider it a blocker, but a nice to have: Ideally, a unit test of `notifySignatureComplete()`. 
I realize it might be tricky to mock the SMTP stuff using approaches like the other tests, e.g.:

`req := httptest.NewRequest...`
but maybe even just a test that validates that error is returned as expect on failure is better than nothing.

Reply:
Test added.